### DISCUSSION TITLE
want to get my engine listed on supported database of superset

### DISCUSSION THREAD

**Discussion by Shubham-kumar-0712** [apache/superset#25145] [Q&A / Help]
Hi,

I have a functional SQLAlchemy dialect and Python driver that works well with superset but now i want my engine to get listed in supported database of superset.
can anyone guide us how to achieve so.

Thank you.

**Comment by sfirke** [comment]
Hello!  What is the database engine?

Here is a how-to blog post about the steps involved in building a new database connector: https://preset.io/blog/building-database-connector/  It's a little old but most of that should still apply.  There you can see things you might need to contribute to the project, like engine specs and tests.

I think if you have completed everything in that blogpost, you could submit a pull request to add it all including into the documentation where it would get listed as a supported database.

I'm going to convert this into a discussion, as it's not a bug report.  If you have further questions feel free to post them here or ask in the project's Slack chat (link on the main Superset readme page).
